modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkStatus.java

```diff
@@ -111,6 +112,9 @@
     @Nullable
     protected Long processingDurationMillis;
 
+    @AvroDefault("false")
+    protected boolean limitedQuery;
```

Comment:
It's not clear to me what his means when it's true. Is it that there was a limit passed when doing the query? Or that the query is incomplete in its results returned.
A useful flag would be more about "limitReached", which I would call "resultsTruncated" or "resultsIncomplete", or "moreResults".

Reply:
limitedQuery is true if the query has a limit and it has been reached which makes sense in the context of the status, the query itself and its attributes (like the limit) are part of the command. limitReached or queryLimitReached could be an alternative.

Reply:
will use queryLimitReached